src/pages/documentation/release-notes/2022.md

```diff
@@ -0,0 +1,459 @@
+---
+description: Archived Release Notes for 2022
+---
+
+# 2022
```

Comment:
This file is the old `index.md` file. I renamed to `2022.md` and edited the description and first header. Everything else in the file is the same.

Reply:
Yup! I figured that's the case, so I didn't bother looking at it since I didn't think you changed anything there :p

Reply:
@kevinlind thanks for this change. Can you also include the 2022.md file and update the index for 2023 in gatsby-config which is used to render the left side nav menu https://github.com/AdobeDocs/aep-mobile-sdkdocs/blob/main/gatsby-config.js#L39 ?